nixos/modules/services/monitoring/prometheus/exporters/script.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.script;
+  configFile = pkgs.writeText "script-exporter.yaml" (builtins.toJSON cfg.settings);
+in
+{
+  port = 9172;
+  extraOpts = {
+    settings = mkOption {
+      type = types.attrs;
+      default = {};
+      example = literalExample ''
+        {
+          scripts = [
+            { name = "success"; script = "sleep 5"; }
+          ];
+        }
+      '';
+      description = ''
+        All settings expressed as an Nix attrset.
+
+        Check the official documentation for the corresponding YAML
+        settings that can all be used here: https://github.com/adhocteam/script_exporter#sample-configuration
```

Comment:
It may be nicer to use something like `<link xlink:href="https://github.com/adhocteam/script_exporter#sample-configuration" />` here. That way the link is clickable in e.g. the HTML manual.